src/popup/modules/UserInterface.js

```diff
@@ -435,6 +437,37 @@ function triggerFileSave(file, filename, requestDownloadPermissions) {
     });
 }
 
+/**
+ * If qrcode text was an URL i.e contains :// this function generates a filename using the domain and replacing all special characters with _
+ *.If not return "qrcode"
+ *
+ * @function
+ * @private
+ * @param { } 
+ * @returns {filename}
+ */
+function generateFilename(){
+	var firstIndex = qrCodeInputText.indexOf("://");
+	if(firstIndex > 0) {
+		// URL found
+		var filename = qrCodeInputText.substring(firstIndex+3);
```

Comment:
```suggestion
		var filename = qrCodeInputText.substring(firstIndex + 3);
```

As per our contributing guide, please use EsLint that checks/enforces such code style things…